socialhome/users/tests/test_views.py

```diff
@@ -387,7 +394,7 @@ def test_get_json_context(self):
         )
         request, view, contents, profile = self._get_request_view_and_content(anonymous_user=True, create_content=False)
         view.followers_count = 3
-        self.assertEqual(
+        self.assertDictEqual(
```

Comment:
This change is unnecessary - `self.assertEqual` calls `self.assertDictEqual` when comparing dict.